From e0af15e6aa74098f46e87f5a8546d4c72522c36a Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH 149/296] UX: Don't draw vertical scrollbar if it is in hidden
 state

If fast scrollbar exit, vertical scrollbar will be hidden,
needn't draw it
---
 core/java/android/view/View.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/java/android/view/View.java b/core/java/android/view/View.java
index 166d6b7a5b1..0b6d046fd88 100644
--- a/core/java/android/view/View.java
+++ b/core/java/android/view/View.java
@@ -15390,7 +15390,8 @@ public class View implements Drawable.Callback, KeyEvent.Callback,
             scrollCache.scrollBar.setCallback(this);
         }
 
-        if (isHorizontalScrollBarEnabled() || isVerticalScrollBarEnabled()) {
+        if (isHorizontalScrollBarEnabled() ||
+                (isVerticalScrollBarEnabled() && !isVerticalScrollBarHidden())) {
 
             if (invalidate) {
                 // Invalidate to show the scrollbars
-- 
2.11.0

